contrib/mixin/mixin.libsonnet

```diff
@@ -227,14 +227,14 @@
           {
             alert: 'etcdExcessiveDatabaseGrowth',
             expr: |||
-              increase(((etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100)[240m:1m]) > 50
+              predict_linear(etcd_mvcc_db_total_size_in_bytes[4h], 4*60*60) > etcd_server_quota_backend_bytes
             ||| % $._config,
             'for': '10m',
             labels: {
               severity: 'warning',
```

Comment:
Does "critical" make more sense than "warning" here, because it needs immediate operator intervention?  @hexfusion @wking @simonpasquier

Reply:
If we're worried about ~hours latencies, then `critical` makes sense, since that's ["immediate action" vs. `warning`'s "office hours queue"][1].  However, part of the current focus on this alert is because the `increase`-on-a-non-counter implementation was creating false-positives, and we risk further alert fatigue if it moves to `critical` and we continue to have false-positives.

Also, In the cases where the admin has clear actions they can take (like compaction/defragmentation), it seems likely that you're growing slowly enough that the `critical` `etcdBackendQuotaLowSpace` kicks in with enough time for you to roll out of bed and run a defrag.  And in cases where it is a new, runaway workload, it seems less likely that a groggy hour or two are enough for you to identify the workload and squash/quota-guard it, and you'll probably end up bumping into the cap and shutting down writes.  So I think the benefit of moving this one to `critical` is small.

I'm not a decision maker here, so feel free to do what feels right to you, but if I was I'd do one round of `expr` fiddling to try to improve signal-to-noise, and then if that seemed to be working well in production, I'd consider a second round of changes pushing the severity up to `critical`.

[1]: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/b8f44bb7be728423836bef0e904ec7166895a34b/README.md#alert-severities